### PR TITLE
Adding definition.json files

### LINKED ISSUE #3: Add missing definition.json files for each standard cell
Each of the standard cells should have a `definition.json` file similar to the skywater-pdk. See example at https://github.com/google/skywater-pdk-libs-sky130_fd_sc_hd/blob/ac7fb61f06e6470b94e8afdf7c25268f62fbd7b1/cells/a2111o/definition.json

### PR DESCRIPTION
**Pull request by mohanad0mohamed**
Fixes #3 

- JSON files are added

- [x] JSON files are added

### CODE REVIEW

**Comment by mohanad0mohamed** [comment]
I have made the same requested changes from the [globalfoundries-pdk-libs-gf180mcu_fd_sc_mcu9t5v0](https://github.com/google/globalfoundries-pdk-libs-gf180mcu_fd_sc_mcu9t5v0) repository here.

**Review by mithro — APPROVED** [review]
Can you please squash the three commits together and `git push --force` this branch. Then I'll merge.

**Comment by mohanad0mohamed** [comment]
Squashed all commits into single one.

File: cells/addf/definition.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
+    "description": "Full Adder, 2X drive strength",
```
Comment (mithro) [inline cells/addf/definition.json]: These shouldn't have the `, 2X drive strength` as there are multiple drive strength versions of the Full adder under the `addf` cell.

**Review by mithro — CHANGES_REQUESTED** [review]

**Comment by mohanad0mohamed** [comment]
Removed all strengths in the description section.

File: cells/addf/definition.json (hunk 1 above)
Comment (mohanad0mohamed) [reply cells/addf/definition.json]: Done

**Review by mithro — APPROVED** [review]
LGTM

**Pull request merged**